.gitignore

```diff
@@ -131,3 +131,7 @@ dmypy.json
 # Docs
 _build
 
+/bin
```

Comment:
Why do we need to add these system folders in our gitignore ? Do you need to add them on your side?

Reply:
I don't remember why. I think it has to do with an automatic generation of this file